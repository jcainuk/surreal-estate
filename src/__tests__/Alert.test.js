import React from "react";
import { render, screen } from "@testing-library/react";
import Alert from "../components/Alert";

describe("Alert", () => {
  test("it displays an error message", () => {
    const { asFragment } = render(<Alert message="Error!" isSuccess={false} />);
    expect(screen.getByText("Error!")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
