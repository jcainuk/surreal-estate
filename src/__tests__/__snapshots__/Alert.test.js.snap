// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert does not render if message prop is empty 1`] = `<DocumentFragment />`;
